homeassistant/components/yeelight/__init__.py

```diff
@@ -287,6 +292,15 @@ def type(self):
 
         return self._device_type
 
+    @property
+    def unique_id(self) -> Optional[str]:
+        """Return a unique ID.
+
+        Uses data returned from get_capabilities call ( https://yeelight.readthedocs.io/en/latest/yeelight.html#yeelight.Bulb.get_capabilities )
```

Comment:
I'm not sure if it's worth pointing the technical details here (or at least the link could be removed).